rllib/connectors/common/add_observations_from_episodes_to_batch.py

```diff
@@ -119,10 +119,7 @@ def __call__(
                 self.add_n_batch_items(
                     data,
                     Columns.OBS,
-                    items_to_add=[
-                        sa_episode.get_observations(indices=ts)
-                        for ts in range(len(sa_episode))
-                    ],
+                    items_to_add=sa_episode.get_observations(slice(0, len(sa_episode))),
```

Comment:
Yup that looks somehow cleaner and faster.

Reply:
Yeah, thanks for mentioning this in the other PR! This helped me pinpoint the performance decrease this time.